documentation/asciidoc/topics/code_examples/queries.js

```diff
@@ -1,9 +1,9 @@
-var infinispan = require('infinispan');
-var protobuf = require('protobufjs');
+const infinispan = require('infinispan');
```

Comment:
I'm not sure about the usage of `const`. I mean: almost all the vars in this example are just assigned once, so for uniformity I would say that they should all be declared as `const`, but this way the result would be pretty weird to me. What do you think?

Reply:
You are right, the only variables that cannot be declared const are `protoMetaClient` and `client`, which are assigned within the try statement and used outside it.
I tried the example using const assignment where was possible to use it, and it's working great